src/libponyrt/lang/stat.c

```diff
@@ -69,6 +71,8 @@ static void windows_stat(const char* path, pony_stat_t* p, struct __stat64* st,
   p->broken = false;
 
   p->hard_links = (uint32_t)st->st_nlink;
+  p->device = (uint32_t)st->st_dev;
```

Comment:
I can't actually image system with 4294967296 devices :).  However no one could imagine so many internet peers when IPv4 was designed

So `uint64_t` is reasonable enough, should I make a change ?